LICENSE

```diff
@@ -202,254 +202,25 @@
-
-com/huawei/flowcontrol/util/DubboUtil.java in this product is copied from org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java and org/apache/skywalking/apm/plugin/dubbo/DubboInterceptor.java of Apache SkyWalking project.
-
-com/huawei/flowcontrol/DubboDefinition.java in this product is copied from org/apache/skywalking/apm/plugin/asf/dubbo/DubboInstrumentation.java and org/apache/skywalking/apm/plugin/dubbo/DubboInstrumentation.java of Apache SkyWalking project.
-
-com/huawei/sermant/sample/servermonitor/provider/OpenJvmMetricProvider.java in this product is copied from org/apache/skywalking/apm/agent/core/jvm/JVMService.java of Apache SkyWalking project.
-
-com/huawei/flowrecord/plugins/dubbo/DubboInterceptor.java in this product is copied from org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java of Apache SkyWalking project.
-
-com/huawei/flowrecord/plugins/dubbo/DubboInstrumentation.java in this product is copied from org/apache/skywalking/apm/plugin/asf/dubbo/DubboInstrumentation.java of Apache SkyWalking project.
-
-com/huawei/flowrecord/plugins/dubbo/AliDubboInterceptor.java in this product is copied from org/apache/skywalking/apm/plugin/dubbo/DubboInterceptor.java of Apache SkyWalking project.
-
-com/huawei/flowrecord/plugins/dubbo/AliDubboInstrumentation.java in this product is copied from org/apache/skywalking/apm/plugin/dubbo/DubboInstrumentation.java of Apache SkyWalking project.
-
-com/huawei/flowcontrol/ApacheDubboInterceptor.java in this product is copied from org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java of Apache SkyWalking project.
-
-com/huawei/flowcontrol/service/ApacheDubboService.java in this product is copied from org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java of Apache SkyWalking project.
-
-com/huawei/flowcontrol/service/ApacheDubboServiceImpl.java in this product is copied from org/apache/skywalking/apm/plugin/asf/dubbo/DubboInterceptor.java of Apache SkyWalking project.
-
-com/huawei/flowcontrol/ApacheDubboDefinition.java in this product is copied from org/apache/skywalking/apm/plugin/asf/dubbo/DubboInstrumentation.java of Apache SkyWalking project.
-
-com/huawei/flowcontrol/AlibabaDubboInterceptor.java in this product is copied from org/apache/skywalking/apm/plugin/dubbo/DubboInterceptor.java of Apache SkyWalking project.
-
-com/huawei/flowcontrol/service/AlibabaDubboService.java in this product is copied from org/apache/skywalking/apm/plugin/dubbo/DubboInterceptor.java of Apache SkyWalking project.
-
-com/huawei/flowcontrol/service/AlibabaDubboServiceImpl.java in this product is copied from org/apache/skywalking/apm/plugin/dubbo/DubboInterceptor.java of Apache SkyWalking project.
-
-com/huawei/flowcontrol/AlibabaDubboDefinition.java in this product is copied from org/apache/skywalking/apm/plugin/dubbo/DubboInstrumentation.java of Apache SkyWalking project.
+Some files in this product were copied from the following project
```

Comment:
keep former statement. Some files in this project contain a portion of the following project and we have declared in each file.